Confuser.Core/Helpers/InjectHelper.cs

```diff
@@ -99,6 +99,9 @@ public static class InjectHelper {
 
 			newMethodDef.Signature = ctx.Importer.Import(methodDef.Signature);
 			newMethodDef.Parameters.UpdateParameterTypes();
+			
+            		foreach (var paramDef in methodDef.ParamDefs)
+               			newMethodDef.ParamDefs.Add(new ParamDefUser(paramDef.Name, paramDef.Sequence));
```

Comment:
Please watch out for the indention. Only a single tab to indent the loop.
In addition the attributes of the Parameters should be copied as well, if we start with that.
```suggestion
               		newMethodDef.ParamDefs.Add(new ParamDefUser(paramDef.Name, paramDef.Sequence, paramDef.Attributes));
```

Reply:
ParamDefs are not required per se and no error appears at runtime ,but they should be included for the sake of completeness